cosim/include/common/bsg_axil.h

```diff
@@ -45,7 +45,7 @@ template <unsigned int W> class pin {
 
   int get() const {
     unsigned int N = 0;
-    for (int i = 0; i < W; i++) {
+    for (int i = 0; i < min(W, (unsigned int)(8*sizeof(unsigned int))); i++) {
```

Comment:
What's going on here?

Reply:
Ah I added this because I created a 64b bus but I was using only 32b data on it. Can revert for this merge.